streams/src/test/java/org/apache/kafka/streams/TopologyTest.java

```diff
@@ -62,6 +62,7 @@
 import static org.junit.Assert.assertThrows;
 import static org.junit.Assert.fail;
 
+@SuppressWarnings("deprecation")
```

Comment:
We should migrate this to use the new non-deprecated methods instead of just suppressing the warning

Reply:
I have created the JIRA to track this https://issues.apache.org/jira/browse/KAFKA-12994